docs/serving/samples/telemetry-go/README.md

```diff
@@ -1,166 +0,0 @@
-This sample runs a simple web server that makes calls to other in-cluster
-services and responds to requests with "Hello World!". The purpose of this
-sample is to show generating [metrics](../../accessing-metrics.md),
-[logs](../../accessing-logs.md) and distributed
-[traces](../../accessing-traces.md). This sample also shows how to create a
-dedicated Prometheus instance rather than using the default installation.
```

Comment:
I'm trying to figure out whether this should stay or go. I think it needs to go -- I don't actually see the description of the "dedicated Prometheus instance" except in the "how to access".